GomokuServer/src/GomokuServer.Api/Program.cs

```diff
@@ -39,4 +45,7 @@
 app.Run();
 
 // Required for functional tests
-public partial class Program { }
+namespace GomokuServer.Api
```

Comment:
Why we need this namespace?

Reply:
sorry, Rider decided to do this automatically, will revert it back